test/TestApps/ToastNotificationsTestApp/main.cpp

```diff
@@ -659,6 +781,13 @@ std::map<std::string, bool(*)()> const& GetSwitchMapping()
         { "VerifyFailedUpdateNotificationDataWithoutPostToast", &VerifyFailedUpdateNotificationDataWithoutPostToast },
         { "VerifyUpdateToastProgressDataUsingEmptyTagAndValidGroup", &VerifyUpdateToastProgressDataUsingEmptyTagAndValidGroup },
         { "VerifyUpdateToastProgressDataUsingEmptyTagAndEmptyGroup", &VerifyUpdateToastProgressDataUsingEmptyTagAndEmptyGroup },
+        { "VerifyExplicitAppId_Unpackaged", &VerifyExplicitAppId_Unpackaged },
+        { "VerifyUnregisterAll_Unpackaged", &VerifyUnregisterAll_Unpackaged },
+        { "VerifyUnregisterAll", &VerifyUnregisterAll },
+        { "VerifyUnregisterTwice", &VerifyUnregisterTwice },
+        { "VerifyUnregisterTwice_Unpackaged", &VerifyUnregisterTwice_Unpackaged },
+        { "VerifyToastProgressDataSequence0Fail", &VerifyToastProgressDataSequence0Fail },
+        { "VerifyToastUpdateZeroSequenceFail", &VerifyToastUpdateZeroSequenceFail_Unpackaged },
```

Comment:
Seems to be a packaged / unpackaged mismatch

Reply:
Yeah im fixing this bug. New PR.